kore/src/Kore/Repl.hs

```diff
@@ -103,7 +103,10 @@ runRepl axioms' claims' logger replScript replMode outputFile = do
         void
             $ flip evalStateT st
             $ flip runReaderT config
-            $ replInterpreter printIfNotEmpty cmd
+            $ replInterpreter
```

Comment:
I think the internals are leaking out a bit here with these changes. How about reverting this change, and renaming `replInterpreter` to `replInterpreter0` or `replInterpreter'` (and renaming all internal occurrences to that), then writing `replInterpreter fn cmd = replInterpreter0 (PrintAuxOutput fn) (PrintKoreOutput fn) cmd`?